sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -391,6 +405,12 @@ export const logger: import("@azure/logger").AzureLogger;
 export interface MergeCertificateOptions extends coreHttp.OperationOptions {
 }
 
+// @public
+export interface OrganizationDetails {
+    adminDetails?: AdministratorContact[];
```

Comment:
`administratorContact`?

Reply:
What do you mean by this?

Reply:
I'm starting to think this was a type we used to use at some point and is not necessary anymore. I'll check tomorrow. Thank you for helping me notice this!

Reply:
Oh! Good news! These types were not necessary. From feedback during the reviews of the public API, we ended up with other types. For example, instead of having this type:

```ts
/**
 * The contacts for the vault certificates.
 */
export interface CertificateContacts {
  /**
   * Identifier for the contacts collection.
   * **NOTE: This property will not be serialized. It can only be populated by the server.**
   */
  readonly id?: string;
  /**
   * The contact list for the vault certificates.
   */
  contactList?: CertificateContact[];
}
```

We ended up deciding on only using `CertificateContact[]`.

Reply:
I'm going to merge this PR. Please let me know if I should follow up with anything else. I appreciate the feedback!